### DISCUSSION TITLE
2 questions

### DISCUSSION THREAD

**Discussion by Heny67** [NickvisionApps/Denaro#810] [General]
Hi!
Where can I enter an induction amount for the group?
How to change the language?

**Comment by nlogozzo** [comment]
> Where can I enter an induction amount for the group?

What do you mean by induction amount? Do you mean like a budget amount that you are allowed to spend every month on a group?

> How to change the language?

Denaro should follow your system language, if translations are available for it. If the app is in English, it means it hasn't been translated for your language yet. What language is your system set to?

**Reply by Heny67** [reply]
"What do you mean by induction amount? Do you mean like a budget amount that you are allowed to spend every month on a group?"

When I create a new group, I would like to enter an initial value that is not 0. E.g.:354,54Ft or 2 486 247Ft

"What language is your system set to?"

Hungarian

**Comment by TerminallyChilll** [comment]
I would also like the ability to create a "starting account balance". Hopefully I'm not overlooking an option. Thank you for your reply in advance and your great work :)
